packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -3,13 +3,11 @@ const globalFetch =
   (typeof window !== 'undefined' && window.fetch) ||
   (typeof global !== 'undefined' && global.fetch);
 
-// eslint-disable-next-line global-require
-const fetch = globalFetch || require('node-fetch');
```

Comment:
try the `require`.call` workaround here as well? then I think you can avoid all api changes, right?